crates/rolldown_plugin_replace/tests/form/assignment/mod.rs

```diff
@@ -0,0 +1,28 @@
+use std::sync::Arc;
+
+use rolldown::BundlerOptions;
+
+use rolldown_plugin_replace::{ReplaceOptions, ReplacePlugin};
+use rolldown_testing::{abs_file_dir, integration_test::IntegrationTest, test_config::TestMeta};
+
+// doesn't replace lvalue in assignment
+// #[tokio::test(flavor = "multi_thread")]
+#[allow(dead_code)]
```

Comment:
You could use `// #[tokio::test(flavor = "multi_thread"), ignore="TODO: implement the feature"]`

Reply:
Anyway, this is small thing.

Reply:
> You could use `// #[tokio::test(flavor = "multi_thread"), ignore="TODO: implement the feature"]`

ok, it is more elegant and I will use it in subsequent PRs